pom.xml

```diff
@@ -375,7 +383,8 @@
         <profile>
             <id>publication</id>
             <properties>
-                <altDeploymentRepository>local::file:./target/staging-deploy</altDeploymentRepository>
+                <altDeploymentRepository>local::file:./target/staging-deploy
+                </altDeploymentRepository>
```

Comment:
Strange formatting 🤔

Reply:
Changed max line length to 120, should be fixed now